visualization/app/codeCharta/util/algorithm/treeMapLayout/__snapshots__/treeMapGenerator.spec.ts.snap

```diff
@@ -1293,7 +1293,7 @@ Array [
       "mcc": 10,
       "rloc": 70,
     },
-    "color": "#DDDDDD",
+    "color": "#69AE40",
```

Comment:
The snapshot change here indicates that there's a change in the functionality. Is this new behavior definitely the expected one?

Reply:
I stumbled over it as well. In https://github.com/MaibornWolff/codecharta/pull/2073/commits/00e4df0b06bce6d4a9ec4a0b9f6b9f6f53898b43 's message I explained 

> refactor: update snapshots
>
> Snapshots included obsolete whitecolor building which weren't activatable in app anymore.
Now snapshots have the default color, which reflects the truth.

Reply:
Same as [this](https://github.com/MaibornWolff/codecharta/pull/2073#discussion_r661286086)